lib/models/task.dart

```diff
@@ -1,18 +1,27 @@
-class Task {
-  String? id;
-  String name;
+import 'package:equatable/equatable.dart';
 
-  Task({id, required this.name});
+class Task extends Equatable {
+  final int? id;
+  final String name;
+
+  const Task({this.id, required this.name});
 
   factory Task.fromJson(Map<String, dynamic> json) {
     return Task(
-      id: json['id'].toString(),
+      id: json['id']
+          as int?, // Check if 'id' is present before converting to String
```

Comment:
This comment looks out of date. I think the comment should say "int" at the end instead of "String"

Reply:
changed